driver/bpf/filler_helpers.h

```diff
@@ -283,7 +283,11 @@ static __always_inline u32 bpf_compute_snaplen(struct filler_data *data,
 		}
 	} else if (data->state->tail_ctx.evt_type == PPME_SOCKET_SENDMSG_X) {
 		struct sockaddr *usrsockaddr;
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(3, 19, 0)
```

Comment:
The awkward thing with RHEL is that it is far from a stock kernel. Does this code apply to all 3.19 kernels or just Red Hat Kernels?

Reply:
All kernels (not that it matters too much for eBPF, but we use the same check in the kmod fillers so I kept it for consistency).